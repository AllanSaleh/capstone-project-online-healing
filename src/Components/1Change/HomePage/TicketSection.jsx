import React from 'react';

import Card from './Card';

const TicketSection = () => (
  <div className="h-section flex flex-col justify-evenly px-sides">
    <h1 className="text-title pb-2 uppercase mb-2">we can communicate through</h1>
    <div className="flex justify-around">
      <Card showIcon={false} title="5 Tickets" content="10$" showButton buttonText="Purchase" />

      <Card showIcon={false} title="25 Tickets" content="40$" showButton buttonText="Purchase" />

      <Card showIcon={false} title="50 Tickets" content="70$" showButton buttonText="Purchase" />
    </div>
  </div>
);

export default TicketSection;
