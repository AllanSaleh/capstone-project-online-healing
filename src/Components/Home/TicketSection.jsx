import React from 'react';
import Card from './Card';

const TicketSection = () => (
  <div className=" md:h-section flex flex-col justify-evenly px-sides">
    <div>
      <h1 className="text-xl md:text-4xl xl:text-title uppercase">
        Purchase tickets
      </h1>
      <h3 className="text-xs md:text-lg lg:text-subtitle text-opacity-75 mb-4 md:my-2">
        purchase tickets that can be used to book appointments!
      </h3>
    </div>
    <div className="flex flex-col md:flex-row items-center md:items-start md:justify-around">
      <Card
        showIcon={false}
        title="5 Tickets"
        content="10$"
        showButton
        buttonText="Purchase"
      />

      <Card
        showIcon={false}
        title="25 Tickets"
        content="40$"
        showButton
        buttonText="Purchase"
      />

      <Card
        showIcon={false}
        title="50 Tickets"
        content="70$"
        showButton
        buttonText="Purchase"
      />
    </div>
  </div>
);

export default TicketSection;
