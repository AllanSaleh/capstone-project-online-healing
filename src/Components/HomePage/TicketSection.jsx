import React from 'react';
import { Link } from 'react-router-dom';

import Card from './Card';

const TicketSection = ({ user }) => (
  <div id="Tickets" className="flex flex-col justify-evenly px-sides lg:h-section py-8">
    <div className="pb-8">
      <h1 className="text-xl md:text-4xl xl:text-title uppercase">Purchase tickets</h1>
      <h3 className="text-xs md:text-lg lg:text-subtitle text-opacity-75 mb-4 md:my-2">
        purchase tickets that can be used to book appointments!
      </h3>
    </div>
    <div className="flex flex-col md:flex-row items-center justify-around w-full lg:h-1/3">
      <Card showIcon={false} title="5" content="10$" showButton buttonText="Purchase" user={user} />

      <Card
        showIcon={false}
        title="25"
        content="40$"
        showButton
        buttonText="Purchase"
        user={user}
      />

      <Card
        showIcon={false}
        title="50"
        content="70$"
        showButton
        buttonText="Purchase"
        user={user}
      />
    </div>
    <div
      className="flex flex-col justify-around items-center p-4 rounded-lg h-1/3 lg:mx-16"
      style={{ boxShadow: '0px 4px 12px 4px rgba(178, 171, 171, 0.25)' }}
    >
      <h1 className="text-2xl lg:text-3xl uppercase text-center py-4">are you a counselor?</h1>
      <p className="w-3/4 text-base md:text-md lg:text-subtitle text-justify md:text-center text-opacity-50">
        Interested in joining our mental health platform? You decide your schedule and how much you
        want to work, weâ€™ll take care of the client referrals and billing details!
      </p>

      <div className="text-center my-8">
        <Link to="/Requirements">
          <button
            type="button"
            className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
          >
            learn more
          </button>
        </Link>
      </div>
    </div>
  </div>
);

export default TicketSection;
