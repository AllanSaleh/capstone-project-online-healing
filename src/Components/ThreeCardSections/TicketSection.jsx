import React from 'react';
import Card from './Card';

const TicketSection = () => (
  <div className="h-section flex flex-col px-sides">
    <h1 className="text-title uppercase">purchase tickets</h1>
    <h2 className="text-subtitle uppercase">
      purchase tickets that can be used to book appointments!
    </h2>
    <div className="flex justify-around mt-28">
      <Card showIcon={false} title="5 Tickets" content="10$" showButton buttonText="Purchase" />

      <Card showIcon={false} title="25 Tickets" content="40$" showButton buttonText="Purchase" />

      <Card showIcon={false} title="50 Tickets" content="70$" showButton buttonText="Purchase" />
    </div>
    <div className="flex flex-col justify-center items-center mt-28">
      <h1 className="text-title uppercase">are you a counselor?</h1>
      <p className="text-paragraph w-2/3 text-center mt-8">
        Interested in joining our mental health platform? You decide your schedule and how much you
        want to work, weâ€™ll take care of the client referrals and billing details!
      </p>
      <button className="text-subtitle uppercase my-8 bg-blue-dark rounded-lg py-2 px-4 border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300">
        learn more
      </button>
    </div>
  </div>
);

export default TicketSection;
