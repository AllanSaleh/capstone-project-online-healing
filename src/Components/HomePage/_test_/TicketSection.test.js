import * as React from 'react';
import TicketSection from "../TicketSection";
import * as renderer from "react-test-renderer";

it('Ticket Section renders correctly', () => {
  const tree = renderer
    .create(<TicketSection/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});