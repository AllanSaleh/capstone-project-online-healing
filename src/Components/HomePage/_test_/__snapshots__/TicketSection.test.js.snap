// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticket Section renders correctly 1`] = `
<div
  className="flex flex-col justify-evenly px-sides"
>
  <div
    className="pb-16 pt-8"
  >
    <h1
      className="text-xl md:text-4xl xl:text-title uppercase"
    >
      Purchase tickets
    </h1>
    <h3
      className="text-xs md:text-lg lg:text-subtitle text-opacity-75 mb-4 md:my-2"
    >
      purchase tickets that can be used to book appointments!
    </h3>
  </div>
  <div
    className="flex flex-col md:flex-row items-center md:items-start md:justify-around"
  >
    <div
      className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
    >
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        5 Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        10$
      </p>
      <button
        className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
        type="button"
      >
        Purchase
      </button>
    </div>
    <div
      className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
    >
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        25 Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        40$
      </p>
      <button
        className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
        type="button"
      >
        Purchase
      </button>
    </div>
    <div
      className="flex flex-col max-w-sm items-center shadow-lg rounded-lg h-full w-11/12 md:w-1/3 my-6 md:my-0 md:mx-2 p-2 lg:py-6 lg:px-12"
    >
      <h3
        className="text-center text-xl  lg:text-3xl 2xl:text-title pb-1 capitalize"
      >
        50 Tickets
      </h3>
      <p
        className="text-opacity-50 text-black text-center text-sm md:text-md lg:text-paragraph py-4"
      >
        70$
      </p>
      <button
        className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
        type="button"
      >
        Purchase
      </button>
    </div>
  </div>
  <div
    className="flex flex-col justify-around my-16 px-4 lg:px-8 rounded-lg"
    style={
      Object {
        "boxShadow": "0px 4px 12px 4px rgba(178, 171, 171, 0.25)",
      }
    }
  >
    <h1
      className="text-2xl md:text-4xl lg:text-title uppercase mx-auto py-4"
    >
      are you a counselor?
    </h1>
    <p
      className="text-sm md:text-md lg:text-subtitle text-justify md:text-center text-opacity-50 my-4 "
    >
      Interested in joining our mental health platform? You decide your schedule and how much you want to work, weâ€™ll take care of the client referrals and billing details!
    </p>
    <div
      className="text-center my-8"
    >
      <button
        className="bg-blue-dark py-2 px-4 uppercase text-paragraph border text-black hover:bg-transparent hover:border-blue-dark hover:text-blue-dark  transition-all duration-200 rounded-md"
        type="button"
      >
        learn more
      </button>
    </div>
  </div>
</div>
`;
